### PR TITLE
Add: Mikrotik ```interface bonding print detail```

### PR DESCRIPTION
**Pull request by kwood92**
Add support for Mikrotik ```interface bonding print detail```

### CODE REVIEW

File: tests/mikrotik_routeros/interface_bonding_print_detail/mikrotik_routeros_interface_bonding_print_detail.raw [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Flags: X - disabled, R - running
+ 0  R ;;; To Cisco Te1/3 - Te1/4
+      name="bond1" mtu=9000 mac-address=4C:5E:0C:14:3F:9D arp=enabled arp-timeout=auto slaves=sfp-sfpplus7,sfp-sfpplus8 mode=802.3ad primary=none link-monitoring=mii arp-interval=100ms arp-ip-targets="" mii-interval=100ms down-delay=0ms up-delay=0ms lacp-rate=1sec transmit-hash-policy=layer-3-and-4 min-links=0
```
Comment (jmcgill298) [inline tests/mikrotik_routeros/interface_bonding_print_detail/mikrotik_routeros_interface_bonding_print_detail.raw:3]:
```suggestion
      name="bond1" mtu=9000 mac-address=4C:5E:0C:14:3F:9D arp=enabled arp-timeout=auto slaves=sfp-sfpplus7,sfp-sfpplus8 mode=802.3ad primary=none link-monitoring=mii arp-interval=100ms arp-ip-targets="" mii-interval=100ms down-delay=0ms up-delay=0ms lacp-rate=1sec transmit-hash-policy=layer-3-and-4 min-links=0

```

File: ntc_templates/templates/mikrotik_routeros_interface_bonding_print_detail.textfsm [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Value Required ID (\d+)
+Value STATUS (X|R)
+Value NAME (\S+)
+Value SLAVES (\S+(?:,)?)
+Value List DESCRIPTION (.+)
```
Comment (jmcgill298) [inline ntc_templates/templates/mikrotik_routeros_interface_bonding_print_detail.textfsm:5]:
```suggestion
Value List DESCRIPTION (.+?)
```

File: tests/mikrotik_routeros/interface_bonding_print_detail/mikrotik_routeros_interface_bonding_print_detail_01.raw [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Flags: X - disabled; R - running
+ 0  R ;;; Link to Mikrotik
+      name="bond1" mtu=1500 mac-address=48:A9:8A:5D:24:05 arp=enabled arp-timeout=auto slaves=sfp-sfpplus9,sfp-sfpplus10 mode=802.3ad primary=none link-monitoring=mii arp-interval=100ms arp-ip-targets="" mii-interval=100ms down-delay=0ms up-delay=0ms lacp-rate=30secs transmit-hash-policy=layer-2 min-links=0
```
Comment (jmcgill298) [inline tests/mikrotik_routeros/interface_bonding_print_detail/mikrotik_routeros_interface_bonding_print_detail_01.raw:3]:
```suggestion
      name="bond1" mtu=1500 mac-address=48:A9:8A:5D:24:05 arp=enabled arp-timeout=auto slaves=sfp-sfpplus9,sfp-sfpplus10 mode=802.3ad primary=none link-monitoring=mii arp-interval=100ms arp-ip-targets="" mii-interval=100ms down-delay=0ms up-delay=0ms lacp-rate=30secs transmit-hash-policy=layer-2 min-links=0

```

**Review by jmcgill298 — CHANGES_REQUESTED** [review]

**Review by jmcgill298 — APPROVED** [review]

File: ntc_templates/templates/mikrotik_routeros_interface_bonding_print_detail.textfsm (hunk 2 above)
Comment (kwood92) [reply ntc_templates/templates/mikrotik_routeros_interface_bonding_print_detail.textfsm:5]: Apologies, I had some local test files that I hadn't yet committed that cover that issue, but the CLI output is similar to when it comes to multiline comments [#1867](https://github.com/networktocode/ntc-templates/pull/1867).

I'm also going to add the \s* to remove leading whitespace too.

File: ntc_templates/templates/mikrotik_routeros_interface_bonding_print_detail.textfsm [hunk 4]
```diff
@@ -0,0 +1,20 @@
+Value Required ID (\d+)
+Value STATUS (X|R)
+Value NAME (\S+)
+Value SLAVES (\S+(?:,)?)
+Value List DESCRIPTION (\s*.+?)
```
Comment (jmcgill298) [inline ntc_templates/templates/mikrotik_routeros_interface_bonding_print_detail.textfsm:5]:
```suggestion
Value List DESCRIPTION (.+?)
```

**Pull request merged**